systemtest/src/test/java/io/strimzi/systemtest/ConnectS2IST.java

```diff
@@ -329,12 +328,19 @@ void testJvmAndResources() {
 
         KafkaConnectS2IUtils.waitForConnectS2IStatus(kafkaConnectS2IName, "Ready");
 
-        String podName = PodUtils.getPodNameByPrefix(kafkaConnectS2IName);
-
-        assertResources(NAMESPACE, podName, kafkaConnectS2IName + "-connect",
-                "400M", "2", "300M", "1");
-        assertExpectedJavaOpts(podName, kafkaConnectS2IName + "-connect",
-                "-Xmx200m", "-Xms200m", "-server", "-XX:+UseG1GC");
+        String podName = kubeClient().listPods("type", "kafka-connect").get(0).getMetadata().getName();
```

Comment:
do we use same type for connect and connects2i ? I think there should be something like `kafka-connect-s2i`

Reply:
in this case you will get exactly one pod - our connects2i pod - but I can change it :)

Reply:
and in config we add this `.addToLabels("type", "kafka-connect")` so thats why I list pods by `kafka-connect`